pyproject.toml

```diff
@@ -262,8 +262,9 @@ module = ['mne.annotations', 'mne.epochs', 'mne.evoked', 'mne.io']
 # -r f (failed), E (error), s (skipped), x (xfail), X (xpassed), w (warnings)
 # don't put in xfail for pytest 8.0+ because then it prints the tracebacks,
 # which look like real errors
-addopts = """--durations=20 --doctest-modules -rfEXs --cov-report= --tb=short \
-    --cov-branch --doctest-ignore-import-errors --junit-xml=junit-results.xml \
+addopts = """--durations=20 --doctest-modules -rfEXs --tb=short \
+    --cov=mne --cov-report=term --cov-branch \
+    --doctest-ignore-import-errors --junit-xml=junit-results.xml \
```

Comment:
it's fine to change this locally if you want to see coverage results in your terminal, but don't *commit* those changes please.  Revert.